app/src/main/kotlin/com/simplemobiletools/applauncher/activities/MainActivity.kt

```diff
@@ -33,6 +33,8 @@ class MainActivity : SimpleActivity(), RefreshRecyclerViewListener {
     private var mStoredPrimaryColor = 0
     private var mStoredTextColor = 0
 
+    private val emptyViews by lazy { arrayOf(add_icons_placeholder, no_items_placeholder) }
```

Comment:
avoid using lazy please, it is unpredictable and hard to read

Reply:
[done!](https://github.com/SimpleMobileTools/Simple-App-Launcher/pull/143/commits/1c6585e5f95ec485a3664c30df8de3b084720636)